src/NServiceBus.Transport.RabbitMQ/RabbitMQTransport.cs

```diff
@@ -39,16 +43,21 @@ public RabbitMQTransport(RoutingTopology routingTopology, string connectionStrin
             ArgumentNullException.ThrowIfNull(connectionString);
 
             RoutingTopology = routingTopology.Create();
-            ConnectionConfiguration = ConnectionConfiguration.Create(connectionString);
+            BrokerConnectionConfiguration = ConnectionConfiguration.Create(connectionString);
+
+            ManagementConnectionConfiguration = string.IsNullOrEmpty(managementConnectionString) ?
+                ConnectionConfiguration.ConvertToManagementConnection(BrokerConnectionConfiguration) :
+                ConnectionConfiguration.Create(managementConnectionString, isManagementConnection: true);
```

Comment:
Oh, yeah my comment about not using `ConnectionConfiguration` above... yeah that needs to change.

The management client connection string is going to be an http/s URL, which isn't something `ConnectionConfiguration` knows how to parse, so we should never be trying to pass that in.

`ConnectionConfiguration` is for parsing our own made up connection string format and the `amqp` format that the client understands.